isb_lib/models/string_list_type.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from sqlalchemy import TypeDecorator, TEXT
+
+
+class StringListType(TypeDecorator):
+    """TypeDecorator that lets us transform an array into a text column.  Works with both SQLite and Postgresql and is
+    also simple and easy to use (unlike the built-in JSON querying mechanism)
+    """
+    impl = TEXT
```

Comment:
@datadavev this is the bit that lets use a text column for a python list of strings in SQLAlchemy